src/SecretsApi/SecretsApi.php

```diff
@@ -22,11 +22,24 @@ class SecretsApi
     private function getBaseURI()
     {
         $config = $this->request()->getConfig();
+
+        $protocol = $config->get('papi_protocol') ?? $config->get('protocol');
+        $port = $config->get('papi_port') ?? $config->get('port');
+        // @todo Change to pantheonapi.svc.pantheon.io once alias is set.
+        $host = 'pantheonapi.production.general-01.us-central1.internal.k8s.pantheon.io';
+        if (!empty($config->get('papi_host'))) {
+            $host = $config->get('papi_host');
```

Comment:
We could reduce the cyclometric complexity of this function if we start by assigning 'papi_host' to $host, and then check $terminusHost iff $host is empty, and then finally plug in the default value if $host is still empty. Same number of `if`s, but we remove an else and one level of nesting.

Reply:
Done. Please take another look